libs/cookies/src/lib/tokens/has-cookies-fallback/has-cookies-fallback.ts

```diff
@@ -0,0 +1,8 @@
+import { InjectionToken, Type } from '@angular/core';
+
+import { NgxCookiesFallBackComponent } from '../../abstracts';
+
+/** A token to provide a fallback component that needs to be rendered when a specific (set of) cookie(s) was not accepted. */
```

Comment:
This is oddly formatted. Any particular reason why?

Reply:
This is simply how you can do it for a single line, it's not better or worse than doing it over multiple lines really.